modules/openy_gc_auth/modules/openy_gc_auth_custom/src/Plugin/GCIdentityProvider/Custom.php

```diff
@@ -67,6 +136,12 @@ public function submitConfigurationForm(array &$form, FormStateInterface $form_s
     if (!$form_state->getErrors()) {
       $this->configuration['enable_recaptcha'] = $form_state->getValue('enable_recaptcha');
       $this->configuration['api_endpoint'] = $form_state->getValue('api_endpoint');
+      $this->configuration['enable_email_verification'] = $form_state->getValue('enable_email_verification');
+      $this->configuration['email_verification_api_endpoint'] = $form_state->getValue('email_verification_api_endpoint');
+      $this->configuration['email_verification_link_life_time'] = $form_state->getValue('email_verification_link_life_time');
+      // TODO: check that text_format works on submit.
```

Comment:
checked?
Is Paige aware and is it in project backlog?

Reply:
Fixed